src/Turing.jl

```diff
@@ -1,202 +1,202 @@
-2. A method of `sample` function that produces results of inference, which is where actual inference happens.
-
-Turing translates models to chunks that call the modelling functions at specified points.
-The dispatch is based on the value of a `sampler` variable.
-To include a new inference algorithm implements the requirements mentioned above in a separate file,
-then include that file at the end of this one.
-"""
-mutable struct Sampler{T, S<:AbstractSamplerState} <: AbstractSampler
-    alg      ::  T
-    info     ::  Dict{Symbol, Any} # sampler infomation
-    selector ::  Selector
-    state    ::  S
-end
-Sampler(alg) = Sampler(alg, Selector())
-Sampler(alg, model::Model) = Sampler(alg, model, Selector())
-Sampler(alg, model::Model, s::Selector) = Sampler(alg, model, s)
-
-include("utilities/Utilities.jl")
-using .Utilities
-include("core/Core.jl")
-using .Core
-include("inference/Inference.jl")  # inference algorithms
-using .Inference
-include("variational/VariationalInference.jl")
-using .Variational
-
-# TODO: re-design `sample` interface in MCMCChains, which unify CmdStan and Turing.
-#   Related: https://github.com/TuringLang/Turing.jl/issues/746
-#@init @require CmdStan="593b3428-ca2f-500c-ae53-031589ec8ddd" @eval begin
-#     @eval Utilities begin
-#         using ..Turing.CmdStan: CmdStan, Adapt, Hmc
-#         using ..Turing: HMC, HMCDA, NUTS
-#         include("utilities/stan-interface.jl")
-#     end
-# end
-
-@init @require DynamicHMC="bbc10e6e-7c05-544b-b16e-64fede858acb" @eval Inference begin
-    using Pkg; 
-    Pkg.installed()["DynamicHMC"] < v"2.0" && error("Please upgdate your DynamicHMC, v1.x is no longer supported")
-    using ..Turing.DynamicHMC: DynamicHMC, mcmc_with_warmup
-    include("contrib/inference/dynamichmc.jl")
-end
-
-# Random probability measures.
-include("stdlib/distributions.jl")
-include("stdlib/RandomMeasures.jl")
-
-###########
-# Exports #
-###########
-
-# Turing essentials - modelling macros and inference algorithms
-export  @model,                 # modelling
-        @VarName,
-
-        MH,                     # classic sampling
-        Gibbs,
-
-        HMC,                    # Hamiltonian-like sampling
-        SGLD,
-        SGHMC,
-        HMCDA,
-        NUTS,
-        DynamicNUTS,
-        ANUTS,
-
-        IS,                     # particle-based sampling
-        SMC,
-        CSMC,
-        PG,
-        PIMH,
-        PMMH,
-        IPMCMC,
-
-        vi,                    # variational inference
-        ADVI,
-
-        sample,                 # inference
-        psample,
-        setchunksize,
-        resume,
-
-        auto_tune_chunk_size!,  # helper
-        setadbackend,
-        setadsafe,
-
-        turnprogress,           # debugging
-
-        Flat,
-        FlatPos,
-        BinomialLogit,
-        VecBinomialLogit,
-        OrderedLogistic,
-        LogPoisson
+        ##############
```

Comment:
I guess the indentation changes are not intended? AFAIK the standard in Julia (and in Turing it seems) is to not indent the top level inside a module.

Reply:
Yes, the standard is no indent.  Maybe this was done automatically by the editor? We also have style guidelines which might be useful to look at: https://turing.ml/dev/docs/contributing/style-guide